modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogServiceImpl.java

```diff
@@ -295,7 +306,7 @@ public CompletableFuture<Void> dropTable(DropTableParams params) {
 
             Arrays.stream(schema.indexes())
                     .filter(index -> index.tableId() == table.id())
-                    .forEach(index -> updateEntries.add(new DropIndexEntry(index.id())));
+                    .forEach(index -> updateEntries.add(new DropIndexEntry(index.id(), index.tableId())));
```

Comment:
Isn't this information excessive?
Does alive index have the "tableId", so that you could read it before deletion?

Reply:
Only the indexId being deleted gets into the event, and to get it you need to refer to the previous version of the catalog, which seems a little more redundant than just passing the tableId.

Reply:
It's not redundant, it's how it's supposed to work. Passing data that's already available elsewhere upon processing is redundant